### PR TITLE
Add a simple ts config file #7

### LINKED ISSUE #7: Add a simple tsConfig file
Add a starter tsconfig file.

### PR DESCRIPTION
**Pull request by Tinet0624**
Closes #7 .

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// RUN THIS ONCE EVRYTIME YOU OPEN PROJECT!
```
Comment (JoeProgrammer88) [inline tsconfig.json:1]: You did not spell "Every time" correctly

File: tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+{
+    "compilerOptions": {
+        "removeComments": true,
+        "outDir": "js",
+        "target": "ES2015",
+        "watch": true
```
Comment (JoeProgrammer88) [inline tsconfig.json:7]: Thanks for adding watch : true! That will help productivity!

**Review by JoeProgrammer88 — CHANGES_REQUESTED** [review]

File: tsconfig.json (hunk 1 above)
Comment (Tinet0624) [reply tsconfig.json]: Fixed it thank you!

**Review by JoeProgrammer88 — APPROVED** [review]
Thanks for making that spelling fix! Looks good!

**Pull request merged**